client/src/controllers/ClipboardController.ts

```diff
@@ -0,0 +1,35 @@
+import { Controller } from '@hotwired/stimulus';
+
+export class ClipboardController extends Controller<HTMLElement> {
```

Comment:
```suggestion
/**
 * Adds the ability for an element to copy the value from a target to the clipboard.
 */
export class ClipboardController extends Controller<HTMLElement> {
```

Nice to have some JSDoc that describes this.